recipe/meta.yaml

```diff
@@ -40,11 +45,12 @@ requirements:
     - c-ares
     - libprotobuf {{ protobuf }}
     - re2
-    - openssl
+    - openssl 1.1.1k  # [x86_64 and c_compiler_version == "7.2.*"]
+    - openssl         # [ppc64le or (x86_64 and c_compiler_version == "7.5.*")]
```

Comment:
Should probably just put: `# [ not (x86_64 and c_compiler_version == "7.2.*")]` here. Otherwise openssl won't appear at all for Z.